be/src/formats/parquet/schema.cpp

```diff
@@ -172,6 +172,8 @@ Status SchemaDescriptor::list_to_field(const std::vector<tparquet::SchemaElement
             // }
             if (is_map(level2_schema)) {
                 RETURN_IF_ERROR(map_to_field(t_schemas, pos + 1, cur_level_info, element, next_pos));
+            } else if (is_list(level2_schema)) {
```

Comment:
If level2_schema could be list, the comment above looks like not accurate?

Reply:
I 'll update this. I don't change comment.